src/govuk/components/character-count/character-count.unit.test.mjs

```diff
@@ -0,0 +1,148 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { createElement } from '../../../../lib/dom-helpers.mjs'
+import CharacterCount from './character-count.mjs'
+
+describe('CharacterCount', () => {
+  describe('formatCountMessage', () => {
+    describe('default configuration', () => {
+      let component
+      beforeAll(() => {
+        // The component won't initialise if we don't pass it an element
+        component = new CharacterCount(createElement('div'))
```

Comment:
Slightly nervous about passing a div here, and imagine this could come back to haunt us in the future when e.g. we get rid of the `init` function (which at the minute we're not calling, so a lot of the 'setup' for the component isn't happening).

Don't have any immediate suggestions for a fix, and this might just be something we have to revisit later… but it feels a bit fragile.

Reply:
Because we have the Puppeteer tests, I'm not too worried about mocking what the component runs onto and skipping its initialisation. 

If anything, having these tests only depend on the bare minimum that the feature needs (one element and no `init` call) can help us ensure we do update tests: if the formatting starts to depend on more than what's mocked, these tests will fail and force us to update them or to better isolate that method, and maybe prompt us to add new cases around the change that we may have missed through the end-to-end tests in Puppeteer (because we can't cover all combinations of everything there, while we can be more thorough in unit tests which run with less overhead). 

Is it the extra updates that worry you?